pkg/webhook/server/server.go

```diff
@@ -59,7 +58,13 @@ func WithCertDir(certDir string) ServerModifier {
 	}
 }
 
-func (s *server) Start(stop <-chan struct{}) error {
+func ServerUpdate(s *Server, serverOpts []ServerModifier) {
```

Comment:
Better use an object receiver
func (s *Server) Server(serverOpts ...ServerModifier)
so we can do 
server.UpdateOptions(webhookserver.WithHook....)